src/fidesops/ops/service/privacy_request/request_runner_service.py

```diff
@@ -233,6 +266,12 @@ async def run_privacy_request(
         logging.info("Dispatching privacy request %s", privacy_request.id)
         privacy_request.start_processing(session)
 
+        manual_data, proceed = get_access_manual_webhook_inputs(
+            session, privacy_request
+        )
+        if not proceed:
+            return
```

Comment:
This is the primary change of this PR - this puts a PrivacyRequest in `requires_input` status if we don't have every manual webhook confirmed.  If we do have confirmation from all our manual webhooks, we retrieve any manually uploaded data, and this is passed onto the `upload_access_results` function and data is combined with the automatically retrieved data at the end.